### DISCUSSION TITLE
Stripe & Supabase Integration Issue at Test Payment Phase

### DISCUSSION THREAD

**Discussion by williamherrmann** [supabase/supabase#16525] [Questions]
Hello, everyone. I've been following this incredible tutorial on setting up a Next.js project integrated with Stripe and Supabase:

https://www.youtube.com/watch?v=7ORL21ZVSI0

I've successfully followed along with no issues through every section until the 1hr 46min mark.

The issue is during the test payment phase, the user information isn't showing in Supabase, like what was demonstrated at the 1hr 46min mark of the video.

However, the test payments and thank you page redirect are working as they should.

In my terminal, I see this error:

"No storage option exists to persist the session, which may result in unexpected behavior when using auth. If you want to set persistSession to true, please provide a storage option or you may set persistSession to false to disable this warning. customer is not defined"

I've triple checked the API details, and the webhook appears to be set up correctly.

I suspect the issue might be somewhere in my code, possibly in the 'webhooks.js' file where we were working during the tutorial.

To make troubleshooting easier for anyone willing to lend a hand, I’ve shared [my public repository here](https://github.com/williamherrmann/epic-saas-from-youtube).

I would greatly appreciate any guidance or suggestions to resolve this issue. I've been enjoying learning how to use supabase and am eager to move past this little problem.

thank you in advance for any help :)

**Comment by GaryAustin1** [comment]
That error message is more a caution than something wrong.  What is saying is you have a supabase client running on a server, or other environment without local storage and persistSession on the client was not set to false in the initialization.  If the client does not use the local storage for saving the user session then everything will still work.

That said, I'm not looking at the repository and don't use next.js or know that tutorial.
